src/hash.h

```diff
@@ -102,13 +102,15 @@ class CHashWriter
     CSHA256 ctx;
 
     const int nType;
-    const int nVersion;
+    int nVersion;
+
 public:
 
     CHashWriter(int nTypeIn, int nVersionIn) : nType(nTypeIn), nVersion(nVersionIn) {}
 
     int GetType() const { return nType; }
     int GetVersion() const { return nVersion; }
+    void SetVersion(int n) { nVersion = n; }
```

Comment:
03572e9 After `git diff c212c7e 703db15` using `OverrideStream`, it looks like we can now revert the changes in `hash.h` and `streams.h`.
```diff
+++ b/src/hash.h
@@ -102,7 +102,7 @@ private:
     const int nType;
-    int nVersion;
+    const int nVersion;
@@ -110,7 +110,6 @@ public:
     int GetVersion() const { return nVersion; }
-    void SetVersion(int n) { nVersion = n; }

+++ b/src/streams.h
@@ -603,7 +603,7 @@ class CAutoFile
     const int nType;
-    int nVersion;
+    const int nVersion; 
@@ -651,7 +651,6 @@ public:
     int GetVersion() const       { return nVersion; }
-    void SetVersion(int n)       { nVersion = n; }
```

Reply:
Good catch! Removed.